README.md

```diff
@@ -7,7 +7,7 @@ You're working on your Tunr app and you encounter this error. What does it mean
 ![Rails error](http://i.imgur.com/9NR7XNT.png)  
 
 ```text
-Your answer...
+When this "template missing" error occurs, it means that the controller did not render a view and did not successfully redirect to the index.html.erb. To fix the problem I would add a create.erb path or might need to define some method or action within the existing index file.
```

Comment:
You identified the correct problem.  We are missing the "app/views/artists/index.html.erb" file.  Your solution has some misconceptions.  I'm not sure how this leads to creating a "create.erb" file.  For two reasons, the first is we need index.html.er, then second is we never (rarely) create a view for create.  Create redirects once it has performed the creation.  Additionally, if the controller needed the index method (or "action"), the error would have indicated that.  Instead, we see a message indicating that it performed the "index" action and now it is attempting to render the "index" form.